persistence-model/src/test/scala/hmda/persistence/serialization/parser/HmdaFileParserProtobufConverterSpec.scala

```diff
@@ -0,0 +1,58 @@
+package hmda.persistence.serialization.parser
+
+import hmda.model.fi.lar.LarGenerators
+import hmda.model.fi.ts.TsGenerators
+import hmda.model.parser.LarParsingError
+import hmda.persistence.messages.events.processing.HmdaFileParserEvents.{ LarParsed, LarParsedErrors, TsParsed, TsParsedErrors }
+import hmda.persistence.model.serialization.HmdaFileParserEvents._
+import hmda.persistence.serialization.parser.HmdaFileParserProtobufConverter._
+import org.scalacheck.Gen
+import org.scalatest.{ MustMatchers, PropSpec }
+import org.scalatest.prop.PropertyChecks
+
+class HmdaFileParserProtobufConverterSpec extends PropSpec with PropertyChecks with MustMatchers
+    with TsGenerators with LarGenerators {
+
+  def listOfStringsGen: Gen[List[String]] = Gen.listOf(Gen.alphaStr)
+  def intGen: Gen[Int] = Gen.choose(0, 100000)
+
+  property("TsParsed must convert to protobuf and back") {
+    forAll(tsGen) { ts =>
+      val message = TsParsed(ts)
+      val protobuf = tsParsedToProtobuf(message).toByteArray
+      tsParsedFromProtobuf(TsParsedMessage.parseFrom(protobuf)) mustBe message
+    }
+  }
+
+  property("TsParsedErrors must convert to protobuf and back") {
+    forAll(listOfStringsGen) { errs: List[String] =>
```

Comment:
It's important to be consistent. Most of our code base does not include types in these property based tests (this is not by accident; most code examples you will find out there look like that). And in your own files within this PR you use it sometimes, but not others, I find that confusing. The type here is self explanatory, it doesn't help with readability. I _know_ it's a list of strings.